src/actions/index.js

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable no-console */
 import * as api from '../services/api';
 
+export const GET_LOCATION_REQUEST = 'GET_LOCATION_REQUEST';
 export const GET_LOCATION_RESPONSE = 'GET_LOCATION_RESPONSE';
+export const GET_LOCATION_RESPONSE_ERROR = 'GET_LOCATION_RESPONSE_ERROR';
```

Comment:
I think usually this would just be "GET_LOCATION_ERROR" (and the other 2 would be "GET_LOCATION_REQUEST" and "GET_LOCATION_SUCCESS").

Reply:
Sounds good. Will fix.

Reply:
Fixed in 86e4e86